bin/label-generator.jl

```diff
@@ -0,0 +1,17 @@
+function main(outfile)
```

Comment:
Do we need this file?

Reply:
Only if you want to regenerate the config once more tracks are added. It's not necessary for the labeler itself.

Reply:
I'll remove it and save it as gist in case it's ever needed.